Management/src/ActuatorWeb/Steeltoe.Samples.ActuatorWeb.csproj

```diff
@@ -0,0 +1,58 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+
+    <!-- Do not override package/assembly/file version with the version from git metadata when using GitInfo. -->
+    <GitVersion>false</GitVersion>
+    <!-- Do not generate (unused) C# classes from git metadata when using GitInfo. -->
+    <GitThisAssembly>false</GitThisAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="idunno.Authentication.Basic" Version="2.4.0" />
+    <PackageReference Include="OpenTelemetry.Exporter.Zipkin" Version="$(OpenTelemetryVersion)" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="$(OpenTelemetryVersion)" />
```

Comment:
`OpenTelemetry.Extensions.Hosting` doesn't appear in the other project file. Removing it compiles without errors. Is it redundant?

Reply:
It is redundant because `Steeltoe.Management.Prometheus` already imports it.